{"uuid":"221be1d2-b12e-4e70-ae20-df83b3f0a3f1","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["db552f43-acf6-44d2-89ad-9a1301b7ae8f"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681570348268,"stop":1681570348269},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681570349144,"stop":1681570349144},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681570349285,"stop":1681570349286},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681570349389,"stop":1681570349389},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681570349427,"stop":1681570349427}],"start":1681570348050,"stop":1681570349428}